server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/BackgroundTask.scala

```diff
@@ -93,6 +95,10 @@ class BackgroundTask(
             .through(metadataCacheAlg.updateCachePipe))
             .compile
             .drain
+            .handleErrorWith {
+              case e =>
+                logger.error(loggingCtx, e)(s"Fail to sync cloud storage")
```

Comment:
this handleErrorWith block is the real change. Basically we shouldn't crash welder when blob can't be found.